### PR TITLE
made some changes to slowmode

### PR DESCRIPTION
**Pull request by Dhruvin12**

### CODE REVIEW

**Comment by sarthak-py** [comment]
could you add something which says that the `slowmode` arg was not provided , since it has the default value as `None` and that wont be acceptable by the `slowmode_delay` method , or just make it a required arg

**Review by sarthak-py — APPROVED** [review]

**Review by kaylebetter — APPROVED** [review]

File: src/cogs/mod.py [hunk 1]
```diff
@@ -111,12 +111,14 @@ async def slowmode(self, ctx: Context, channel: disnake.TextChannel = None, slow
...
         if not channel:
             channel = ctx.channel
-            
+
         if slowmode == 0:
             await channel.slowmode_delay(slowmode)
             return await ctx.send(
                 f"{ctx.author.mention}, removed slowmode in {channel.mention}."
             )
+        elif not slowmode:
```
Comment (Yat-o) [inline src/cogs/mod.py:120]: cxt??????

File: src/cogs/mod.py [hunk 2]
```diff
@@ -109,10 +109,13 @@ async def unban(
     async def slowmode(self, ctx: Context, channel: disnake.TextChannel = None, slowmode: int = None):
         """Change/disable slowmode in a channel"""
 
-        channel = channel or ctx.channel
-        if not slowmode:
+        if not channel:
+            channel = ctx.channel
```
Comment (Yat-o) [inline src/cogs/mod.py:113]: `channel = channel or ctx.channel` would be better.

File: src/cogs/mod.py (hunk 1 above)
Comment (Yat-o) [inline src/cogs/mod.py:120]: Make it required. And let the global error handler deal with it.

**Review by Yat-o — COMMENTED** [review]
Fix everything I said before we merge.

**Pull request merged**